api/swagger.yaml

```diff
@@ -4929,6 +4929,11 @@ paths:
           description: "Only return logs since this time, as a UNIX timestamp"
           type: "integer"
           default: 0
+        - name: "until"
+          in: "query"
+          description: "Only return logs before this time, as a UNIX timestamp"
+          type: "integer"
+          default: 0
```

Comment:
default value is 0?
--since and --until have the same default value 0, I think there exists incompatibility.

Reply:
Yes, they're effectively just placeholders which indicate no limits. What other int do you suggest we can use that's better than `0`?